operator.go

```diff
@@ -656,23 +656,27 @@ func (o *operator) runInternal(ctx context.Context) error {
 					jj := j
 					o.store.loopIndex = &jj
 					if err := stepFn(o.thisT); err != nil {
+						lc := *o.store.loopIndex
 						o.store.loopIndex = nil
-						return err
+						return fmt.Errorf("loop failed (count: %d/%d): %w", lc, c, err)
```

Comment:
@k2tzumi How about an error message like this?

Reply:
I think it's very good.
Let me verify the contents of the loop counter display.